docs/how_tos/i18n.rst

```diff
@@ -116,84 +116,88 @@ The pipeline jobs live in the ``ecommerce-scripts`` repo, but you don't have to
             "i18n": {
               "plugins": [
                 ["react-intl", {
-                  "messagesDir": "./temp"
+                  "messagesDir": "./babel-plugin-react-intl-temp",
+                  "moduleSourceName": "@edx/frontend-i18n"
                  }]
                ]
              }
            }
 
-      * Confirm that running ``make i18n.extract`` creates a lot of ``.json`` files in ``your-repo/temp/`` .
+      * Confirm that running ``make i18n.extract`` creates a lot of ``.json`` files in ``your-repo/babel-plugin-react-intl-temp/`` .
 
-      * Add ``temp`` to your ``.gitignore``.
+      * Add ``babel-plugin-react-intl-temp`` and ``src/i18n/transifex_input.json`` to your ``.gitignore``.
```

Comment:
I was considering changing this to `./temp/babel-plugin-react-intl` which felt more canonical to me, hiding it under `temp` and all.  I recall that it used to be in `temp`, of course... does moving it to a sub-dir work, from your perspective?  I thought I heard the fear was that other things were also using the directory, perhaps.